package.json

```diff
@@ -25,7 +25,7 @@
     "bfx-hf-ext-plugin-dummy": "github:bitfinexcom/bfx-hf-ext-plugin-dummy#v1.0.4",
     "bfx-hf-models": "git+https://github.com/bitfinexcom/bfx-hf-models.git#v2.4.1",
     "bfx-hf-models-adapter-lowdb": "git+https://github.com/bitfinexcom/bfx-hf-models-adapter-lowdb.git#v1.0.5",
-    "bfx-hf-token-renewal-plugin": "git+https://github.com/bitfinexcom/bfx-hf-token-renewal-plugin.git#v1.0.0",
+    "bfx-hf-token-renewal-plugin": "git+https://github.com/tarcisiozf/bfx-hf-token-renewal-plugin.git#handle-token-exception",
```

Comment:
wait for merge

Reply:
this shouldn't happen here? what should be merged on renewal plugin?

Reply:
merge and relese: https://github.com/bitfinexcom/bfx-hf-token-renewal-plugin/pull/2